package-lock.json

```diff
@@ -10,7 +10,7 @@
       "integrity": "sha512-OfC2uemaknXr87bdLUkWog7nYuliM9Ij5HUcajsVcMCpQrcLmtxRbVFTIqmcSkSeYRBFBRxs2FiUqFJDLdiebA==",
       "dev": true,
       "requires": {
-        "@babel/highlight": "^7.0.0"
+        "@babel/highlight": "7.0.0"
```

Comment:
do you know why all of the package-lock.json file changed? Might be difference in version of npm that we're using? I'm using npm@6.9.0

Reply:
I'm using 5.5.1, I'll upgrade

Reply:
and discard changes to this file and package lock